package.json

```diff
@@ -13,12 +13,11 @@
     "@deck.gl/google-maps": "^8.9.15",
     "@deck.gl/layers": "^8.9.15",
     "@deck.gl/mesh-layers": "^8.9.15",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
     "@mui/icons-material": "^5.11.16",
     "@mui/lab": "^5.0.0-alpha.125",
     "@mui/material": "^5.11.16",
-    "@emotion/react": "^11.10.6",
-    "@emotion/styled": "^11.10.6",
-    "styled-components": "^5.2.3",
```

Comment:
I removed this unnecessary reference. We're using emotion styled api, not styled-components